go.mod

```diff
@@ -13,8 +13,8 @@ require (
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/tools v0.1.6-0.20210820212750-d4cc65f0b2ff
-	k8s.io/apimachinery v0.23.0 // for `kubebuilder alpha config-gen`
-	sigs.k8s.io/controller-runtime v0.11.0
+	k8s.io/apimachinery v0.23.5 // for `kubebuilder alpha config-gen`
+	sigs.k8s.io/controller-runtime v0.11.2
```

Comment:
@camilamacedo86 

> I think the projects /scaffold will only change the dep when we bump controller runtime.
> So, we need to bump the controller-runtime as a follow up to close the issue.

This should do this job right ? It was in this first commit too.